src/lottery/services/publicNotice.js

```diff
@@ -115,10 +165,25 @@ const getTicketLeaderboardHandler = async (req, res) => {
     if (rank >= 0)
       res.json({
         leaderboard,
+        totalTicket1Amount,
+        totalTicket2Amount,
+        totalUserAmount: users.length,
         rank: rank + 1,
         probability: sortedUsers[rank].weight / weightSum,
+        probabilityV2: calculateProbabilityV2(
+          users,
+          weightSum,
+          base,
+          sortedUsers[rank].weight
+        ),
+      });
+    else
+      res.json({
+        leaderboard,
+        totalTicket1Amount,
+        totalTicket2Amount,
+        totalUserAmount: users.length,
       });
-    else res.json({ leaderboard });
```

Comment:
```suggestion
    res.json({
      leaderboard,
      totalTicket1Amount,
      totalTicket2Amount,
      totalUserAmount: users.length,
      ...(rank >= 0 ? {
        rank: rank + 1,
        probability: sortedUsers[rank].weight / weightSum,
        probabilityV2: calculateProbabilityV2(
          users,
          weightSum,
          base,
          sortedUsers[rank].weight
        ),
      } : {}),
    });
```

로 하면 겹치는 코드가 줄어들 것 같습니다 ~

Reply:
오... 대박이네요! 감사합니다!